server/bin/immich-healthcheck

```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env bash
 
-node /usr/src/app/dist/bin/healthcheck.js
+if [[ "$IMMICH_WORKERS_EXCLUDE" == *"api"* ]] ||
+   [[ "$IMMICH_WORKERS_INCLUDE" != *"api"* ]]; then
+        echo "API worker excluded, skipping";
+        exit 0;
+fi
+
+IMMICH_HOST="${IMMICH_HOST:=localhost}"
+IMMICH_PORT="${IMMICH_PORT:=2283}"
+
+result=$(curl -m 2 http://"$IMMICH_HOST":"$IMMICH_PORT"/api/server/ping)
```

Comment:
would probably use `curl -fsS` here, because we don't need all the stderr output of curl.